core/src/main/scala/org/apache/spark/api/python/PythonRDD.scala

```diff
@@ -374,49 +375,63 @@ private[spark] object PythonRDD extends Logging {
     // The right way to implement this would be to use TypeTags to get the full
     // type of T.  Since I don't want to introduce breaking changes throughout the
     // entire Spark API, I have to use this hacky approach:
+    def write(bytes: Array[Byte]) {
+      if (bytes == null) {
+        dataOut.writeInt(SpecialLengths.NULL)
+      } else {
+        dataOut.writeInt(bytes.length)
+        dataOut.write(bytes)
+      }
+    }
+
+    def writeS(str: String) {
```

Comment:
Maybe call this `writeString`?  It's a little bit more verbose, but less prone to typos.